lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -82,6 +83,24 @@ def main():
     else:
         config = std_config
 
+    increase_nsb = False
+    increase_psf = False
+    if "image_modifier" in config:
+        imconfig = config["image_modifier"]
+        increase_nsb = imconfig["increase_nsb"]
+        increase_psf = imconfig["increase_psf"]
+        if increase_nsb or increase_psf:
+            log.info(f"image_modifier configuration: {imconfig}")
+        extra_noise_in_dim_pixels = imconfig["extra_noise_in_dim_pixels"]
+        extra_bias_in_dim_pixels = imconfig["extra_bias_in_dim_pixels"]
+        transition_charge = imconfig["transition_charge"]
+        extra_noise_in_bright_pixels = imconfig["extra_noise_in_bright_pixels"]
+        smeared_light_fraction = imconfig["smeared_light_fraction"]
+        if (increase_nsb or increase_psf) and args.noimage is False:
+            log.info("NOTE: Using the image_modifier options means images will "
+                     "not be saved.")
+            args.noimage = True
```

Comment:
Why force disable this? What If we want to check if the code is working as expected by storing the images and comparing them to real data?

Reply:
I was afraid of double-application... Any proposal for a safeguard against that?

Reply:
store the used values in the metadata and fail to apply that is already there in the input file?

Reply:
> I'd say it is not urgent. 

Then why actively disable a feature that is already implemented? Especially one that would be quite handy to have to check how good this feature is?

Reply:
The way it is done is that the whole 'images' thing is removed if not wanted in the output.   So it is not that I removed a feature, the image was never "actively saved" by the script. It is probably easy to do, but not for me, and since I did not need that feature, and time is limited, I did not add it.  It can be done later when one needs it. If you need it now, please go ahead, I am not against it.

Reply:
> I meant testing in the sense of comparing e.g. pixel distributions between data and mc.
> 
> For that, being able to load an image table transformed like this would be helpful, right?

I've done that applying the correction on the fly when you do the comparison.

Reply:
ah ok, sorry, forgot that the dl1ab script only copies the intial table. Fine with me.

Reply:
Ok, thanks!